scripts/weblate/weblate_locks_notify_community.py

```diff
@@ -40,4 +50,4 @@
         print(f'Other error occurred while notifying Mattermost-channel: {err.message} {err.args}')
     else:
         print('Success!')
-json.dump(current_locks,open("locks.txt",'w'))
\ No newline at end of file
+json.dump(current_locks,open("/home/tom/scripts/mattermost/locks.txt",'w'))
```

Comment:
Same concern as previous: will the `/home/tom/scripts/mattermost` path exist on the Weblate server? If not, this path may need to change.